content/upcoming/announcing_babelfish.md

```diff
@@ -0,0 +1,82 @@
+---
+author: abeaumont
+date: 2017-06-06
+title: "Announcing Babelfish"
+draft: true
+image: **TODO: Babelfish image (/post/announcing-babelfish/babelfish.jpg)**
+description: "Announcing Babelfish, the project we are developing to build
+representations of source code."
+categories: ["technical"]
+---
+
+At source{d} we believe there's a possibility for programs to write their own
```

Comment:
I see the ambiguity that @serabe mentions, "to write code" will make it clearer and broader. Thanks for catching my mistake.